src/shared/particle_dynamics/solid_dynamics/thin_structure_dynamics.cpp

```diff
@@ -266,73 +266,6 @@ void ConstrainShellBodyRegionAlongAxis::update(size_t index_i, Real dt)
     dangular_vel_dt_[index_i][1 - axis_] = 0.0;
 }
 //=================================================================================================//
-DistributingPointForcesToShell::
-    DistributingPointForcesToShell(SPHBody &sph_body, std::vector<Vecd> point_forces,
-                                   std::vector<Vecd> reference_positions, Real time_to_full_external_force,
-                                   Real particle_spacing_ref, Real h_spacing_ratio)
-    : LocalDynamics(sph_body), DataDelegateSimple(sph_body),
-      point_forces_(point_forces), reference_positions_(reference_positions),
-      time_to_full_external_force_(time_to_full_external_force),
-      particle_spacing_ref_(particle_spacing_ref), h_spacing_ratio_(h_spacing_ratio),
-      pos_(*particles_->getVariableByName<Vecd>("Position")),
-      force_prior_(*particles_->getVariableByName<Vecd>("ForcePrior")),
-      thickness_(*particles_->getVariableByName<Real>("Thickness"))
-{
-    for (size_t i = 0; i < point_forces_.size(); i++)
-    {
-        weight_.push_back(StdLargeVec<Real>(0.0));
-        time_dependent_point_forces_.push_back(Vecd::Zero());
-        sum_of_weight_.push_back(0.0);
-        particles_->registerVariable(weight_[i], "Weight_" + std::to_string(i));
```

Comment:
The variable name "Weight_" seems not follow the previous style.

Reply:
OK. I am not satisfied with this method, and thinking that should chant it entirely.